plugins/inputs/kernel/kernel_test.go

```diff
@@ -4,12 +4,37 @@ package kernel
 
 import (
 	"testing"
+	"time"
 
-	"github.com/stretchr/testify/require"
-
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
 	"github.com/influxdata/telegraf/testutil"
+
+	"github.com/stretchr/testify/require"
```

Comment:
Please keep the order. We usually have three groups (_built-in_, _3rd-party_ and _telegraf internal_ packages) that are sorted alphabetically within their group. The groups a separated by a blank line.